2022-05-29 17:33:48,151 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-05-29 17:33:48,151 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Python39\lib\site-packages\instabot\bot\bot.py
2022-05-29 17:33:48,152 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-05-29 17:33:48,851 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2022-05-29 17:33:48,851 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-05-29 17:33:48,851 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-05-29 17:33:50,740 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-05-29 17:33:51,814 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-05-29 17:33:53,574 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [429]>
2022-05-29 17:33:53,574 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2022-05-29 17:33:53,574 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2022-05-29 17:33:56,931 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-05-29 17:33:56,931 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXFiEQJnKjroUkMDbmztM9FYuv4FxnzxiH_nQd3ECbzlV6c3joJeGDz3GppU9FuAdGErCw8/bd6YzJiCfnp2Oekt/","api_path":"/challenge/AXFiEQJnKjroUkMDbmztM9FYuv4FxnzxiH_nQd3ECbzlV6c3joJeGDz3GppU9FuAdGErCw8/bd6YzJiCfnp2Oekt/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-05-29 17:33:56,931 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-05-29 17:33:56,931 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-05-29 17:33:56,932 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
